sirepo/pkcli/jupyterhublogin.py

```diff
@@ -58,9 +60,21 @@ def maybe_create_sirepo_user(qcall, email, display_name):
         with qcall.auth.logged_in_user_set(u, method=sirepo.auth.METHOD_EMAIL):
             n = sirepo.sim_api.jupyterhublogin.create_user(
                 qcall=qcall,
-                check_dir=True,
             )
             qcall.auth_db.model("UserRole").add_roles(
                 roles=[sirepo.auth_role.for_sim_type("jupyterhublogin")],
             )
         return PKDict(email=email, jupyterhub_user_name=n)
+
+
+def old_user_dirs():
+    """Outputs commands to delete dirs for Jupyter users that are not in database."""
+    with sirepo.quest.start() as qcall:
+        m = ""
+        r = qcall.auth_db.model("JupyterhubUser").search_all_for_column("user_name")
+        for d in pkio.sorted_glob(
+            sirepo.sim_api.jupyterhublogin.cfg().user_db_root_d.join("*")
+        ):
+            if not os.path.basename(d) in r:
+                m += f"\nrm -rf {d}"
+        pkdlog(m)
```

Comment:
you can just `return m`. This way the function is more flexible. It can be called in python and get a useful result back or on the cli and pkcli does the work of converting the returned value to a printable value.

Reply:
[ad6298a](https://github.com/radiasoft/sirepo/pull/7027/commits/ad6298ae14d580069dd27cda09abc4bfad79f545)